src/Management/src/Tracing/TracingLogProcessor.cs

```diff
@@ -26,41 +29,34 @@ public string Process(string message)
 
         TelemetrySpan? currentSpan = GetCurrentSpan();
 
-        if (currentSpan != null)
+        if (currentSpan == null)
         {
-            TracingOptions options = _optionsMonitor.CurrentValue;
-
-            SpanContext context = currentSpan.Context;
-
-            var sb = new StringBuilder(" [");
-            sb.Append(options.Name);
-            sb.Append(',');
+            return message;
+        }
 
-            string traceId = context.TraceId.ToHexString();
+        SpanContext context = currentSpan.Context;
```

Comment:
`UseShortTraceIds` is available in the Zipkin exporter, but I think it's a pretty legacy option at this point and all it does is take a substring of the trace Id. Also, since we're not configuring the Zipkin exporter anymore, I think it's better to just log the whole id and keep it simpler